### ISSUE TITLE
start.vbs not doing anything

### ISSUE THREAD

**Issue by TubbyHS** [MistyAI/MistyFN#8]
Hi, I've checked the solved issues and found someone else who had this issue. Like he did, I followed the instructions on the page and installed the requirements via install_requirements.bat, installed python 3.8, installed visual studio 2022 with the python addon and installed the GeForce Game Ready Driver(also tried without installing the game ready driver), but start.vbs does nothing. Any idea how I could fix this? I saw the person who previously had this problem fixed the issue by manually installing the requirements, so I have tried just installing the requirements.txt file without the bat file but still no luck. Any help would be appreciated, this project looks awesome and I would love to be able to try it out. I am running Windows 10 fully updated if that means anything.  Thanks for your patience if you've read this far.

**Comment by TubbyHS** [comment]
It somehow resolved itself.

**Comment by Allmight0000** [comment]
I have the same problem here

**Comment by Allmight0000** [comment]
> It somehow resolved itself.


can you help me solve it?

**Comment by TubbyHS** [comment]
> > It somehow resolved itself.
> 
> can you help me solve it?
I can't give you 100% chance solution, but this is what I did.
One of the first things I did was make a .bat file with the contents of : 
```
@echo off
python src/main.py
pause
```

I use that instead of the VBScript file, works much better for me and gives more information as to what you're missing and what not. Run the program and if you're missing anything it should tell you, manually install everything it says you require via pip in command prompt.
Make sure you have discord install and logged in, as the script uses discord services.
Good luck
